.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 node_modules/
 .env
+*~
```

Comment:
What's that

Reply:
Backup files created by "vi" editor. Prett standard in .gitignores where I come from :D